modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/checkpoint/CheckpointHistory.java

```diff
@@ -49,6 +50,24 @@
  * This directory holds files for checkpoint start and end.
  */
 public class CheckpointHistory {
+    /** The message appears when no one checkpoint was not reserved for cache. */
+    private static final String NOT_RESERVED_WAL_REASON = "If checkpoint WAL history can't be reserved";
+
+    /** The message puts down to log when an exception happened during reading reserved WAL. */
+    private static final String WAL_SEG_CORRUPTED_REASON = "Segment corrupted";
+
+    /** Reason means no more history reserved for the cache. */
+    private static final String NO_MORE_HISTORY_REASON = "No more history for reservation";
+
+    /** Node does not have owning partitions. */
+    private static final String NO_PARTITIONS_OWNED_REASON = "No own partitions";
```

Comment:
Node didn't own any partitions for this group at the time of checkpoint

Reply:
If we will be absolutely honest, then we can have own partition, but the partition has  fewer entries that IGNITE_PDS_WAL_REBALANCE_THRESHOLD.